portability-gateway/src/main/java/org/dataportabilityproject/gateway/crypto/EncrypterFactory.java

```diff
@@ -0,0 +1,36 @@
+package org.dataportabilityproject.gateway.crypto;
+
+import com.google.common.base.Preconditions;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import javax.crypto.SecretKey;
+
+public class EncrypterFactory {
```

Comment:
nit: documentation since it's a public class

Reply:
Done. Good point.